.ruby-version

```diff
@@ -1 +1 @@
-2.4.4
+2.7.2
```

Comment:
Will this change made mandatory the use of ruby 2.7.2 for everyone using the gem?

Reply:
Did some reading on this. This file has no impact on the use of this gem. It is mainly for local development. We can specify a `required_ruby_version` in Gemspec which will impact everyone.

I noticed [support for ruby 2.4. has ended](https://www.ruby-lang.org/en/news/2020/04/05/support-of-ruby-2-4-has-ended/), so thought I would upgrade the development requirement.

Debatable whether this PR is really needed 😄